app/assets/stylesheets/components/_accordion.scss

```diff
@@ -20,25 +20,45 @@
     opacity: 1;
   }
 
-  .accordion-header-control {
-    background-position: right .25rem center;
-    background-repeat: no-repeat;
-    background-size: $h4;
-    color: $blue;
+  .accordion-header {
     cursor: pointer;
+    position: relative;
 
-    &[aria-expanded="true"] {
-      background-image: url(image-path('minus.svg'));
+    img {
+      position: absolute;
+      right: 1rem;
+      top: .8rem;
     }
+  }
+
+  .accordion-header-control[aria-expanded="false"] {
+    .minus-icon {
+      display: none;
+    }
+
+    .plus-icon {
+      display: initial;
+    }
+  }
+
+  .accordion-header-control[aria-expanded="true"] {
+    .minus-icon {
+      display: initial;
+    }
+
+    .plus-icon {
+      display: none;
+    }
+  }
+
+  .accordion-header-control {
+    color: $blue;
+    margin-right: 30px;
```

Comment:
Ok, really nitpicking here, but wondering if we could switch this to `padding-right: 30px` and switching the `pl2` on `.accordion-header` to `px2` in order to give us an accurate focus outline like this:

![screen shot 2017-04-04 at 2 19 48 pm](https://cloud.githubusercontent.com/assets/1178494/24672270/2cd3fcdc-1942-11e7-94b3-afa796286b4b.png)

rather than:
![screen shot 2017-04-04 at 2 20 01 pm](https://cloud.githubusercontent.com/assets/1178494/24672275/3142b2ae-1942-11e7-82a0-b33e55b63cda.png)